content/en/Integrations/IntegrationBuilder/How to Guides/github.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "GitHub"
+linkTitle: "GitHub"
```

Comment:
What do you think of naming the article "Push findings to GitHub" instead of just "GitHub"?